pkg/sql/create_table.go

```diff
@@ -246,6 +247,16 @@ func hasPrimaryKeySerialType(params runParams, colDef *tree.ColumnTableDef) (boo
 }
 
 func (n *createTableNode) startExec(params runParams) error {
+	// Lease the system database to see if schema changes are blocked on reader
```

Comment:
could you instead just lease the parent database? but maybe it doesn't matter. Same q for `create_view`

Reply:
not sure why `createTable` and `createView` are special cased, but i trust you 😁